### PR TITLE
Document new Ruby and PHP versions

### LINKED ISSUE #216: Document new Ruby versions that are available and how to use them
@bogyo210 please share list of available versions.

### LINKED ISSUE #218: Document new PHP versions that are available and how to use them
@bogyo210 please provide list of available versions.

### PR DESCRIPTION
**Pull request by mactsouk**
Closes #218  
Closes #216

### CODE REVIEW

File: ubuntu-1804-image_5b461f730428630abc0bf442.md [hunk 1]
```diff
@@ -69,32 +69,35 @@ Docker toolset is installed and following versions are available:
...
 ### Erlang and Elixir
 
-Erlang versions are installed and managed via
-[kerl](https://github.com/kerl/kerl). Elixir versions are installed with
-[kiex](https://github.com/taylor/kiex).
+Erlang versions are installed and managed via [kerl](https://github.com/kerl/kerl).
+Elixir versions are installed with [kiex](https://github.com/taylor/kiex).
 
-- Erlang: 21.0
+- Erlang: 20.0, 21.0
```
Comment (bogyo210) [inline ubuntu-1804-image_5b461f730428630abc0bf442.md]: the erlag 20.0 is actually 20.3

File: ubuntu-1804-image_5b461f730428630abc0bf442.md [hunk 2]
```diff
@@ -139,8 +142,9 @@ Installed version:
 PHP versions are managed by [phpbrew](https://github.com/phpbrew/phpbrew).
 Installed versions:
 
-- 7.0.31
-- 7.1.21
+- 5.6.38
+- 7.0.32
+- 7.1.23
 - 7.2.10
```
Comment (bogyo210) [inline ubuntu-1804-image_5b461f730428630abc0bf442.md]: php default installed version is 7.2.10

Comment (bogyo210) [inline ubuntu-1804-image_5b461f730428630abc0bf442.md]: php installed default version is 7.2.10

File: ubuntu-1804-image_5b461f730428630abc0bf442.md (hunk 1 above)
Comment (mactsouk) [reply ubuntu-1804-image_5b461f730428630abc0bf442.md]: OK - I am changing that to 20.3. Thanks!

**Comment by markoa** [comment]
Made Ruby list shorter.

**Pull request merged**